site/en/gemini-api/docs/function-calling/python.ipynb

```diff
@@ -131,7 +131,6 @@
         "import time\n",
```

Comment:
<blockquote>The&nbsp;<code>google.ai.generativelanguage</code>&nbsp;client library</blockquote><p> </p><p>Are you cleaning these up too? Or just protos / glm directly?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/google/generative-ai-docs/pull/446/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='35'/>

Reply:
<p>Fixed.</p>